### PR TITLE
Move file status icon to the left + spacing adjustments

### LINKED ISSUE #58: status and action buttons look very similar
The error icon (red) and checkmark icon (black) look similar to the cancel and restart icon buttons. Some users thought that they’re also clickable.

Suggestion done during the [UX testing](https://docs.google.com/document/d/12B7KE2PbWBKHO0JakMRcePo6bfD3jn5oiz0XrVl7FgI)

### LINKED ISSUE #60: restart and cancel button too close
The restart file button is too close to the cancel button. On mobile, it’s too easy to miss and remove the file.

Suggestion done during the [UX testing](https://docs.google.com/document/d/12B7KE2PbWBKHO0JakMRcePo6bfD3jn5oiz0XrVl7FgI)

### PR DESCRIPTION
**Pull request by platosha**
Fixes #60, fixes #58

---
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-upload/75)

### CODE REVIEW

**Comment by manolo** [comment]
Review status: 0 of 3 files reviewed at latest revision, 2 unresolved discussions.

---

*[vaadin-upload-file.html, line 18 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/75#-KDOc_iKWkXHdoC3S8mw:-KDOc_iKWkXHdoC3S8mx:1420124579) ([raw file](https://github.com/vaadin/vaadin-upload/blob/df3fdbadbf5014b400ba417d5b2c591dde60cfd1/vaadin-upload-file.html#L18)):*
The second sentence is unclear. I would remove it, or just say: By default, the file icon is invisible until the upload process finishes.

---

*[vaadin-upload-file.html, line 19 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/75#-KDOdgppj4VvZst2-UQI:-KDOdgppj4VvZst2-UQJ:1792411214) ([raw file](https://github.com/vaadin/vaadin-upload/blob/df3fdbadbf5014b400ba417d5b2c591dde60cfd1/vaadin-upload-file.html#L19)):*
I would change -visible for something more meaningful  like -done.
The second sentence could be removed.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-upload/75)*

**Comment by manolo** [comment]
Review status: 0 of 3 files reviewed at latest revision, 3 unresolved discussions.

---

*[vaadin-upload-file.html, line 199 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/75#-KDOfXd6dKg0UFxLCQRN:-KDOfXd6dKg0UFxLCQRO:88549345) ([raw file](https://github.com/vaadin/vaadin-upload/blob/df3fdbadbf5014b400ba417d5b2c591dde60cfd1/vaadin-upload-file.html#L199)):*
why not use iron-icon instead of paper-icon-button 

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-upload/75)*

**Comment by samiheikki** [comment]
Reviewed 3 of 3 files at r1.
Review status: all files reviewed at latest revision, 4 unresolved discussions.

---

*[vaadin-upload-file.html, line 18 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/75#-KDOc_iKWkXHdoC3S8mw:-KDSCZscB7TvHmabBhjw:309157622) ([raw file](https://github.com/vaadin/vaadin-upload/blob/df3fdbadbf5014b400ba417d5b2c591dde60cfd1/vaadin-upload-file.html#L18)):*
Seconded.

---

*[vaadin-upload-file.html, line 63 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/75#-KDSErPLMmg1BzPg7Vkg:-KDSDgIwJfGVrq9qKfZf:-420820679) ([raw file](https://github.com/vaadin/vaadin-upload/blob/df3fdbadbf5014b400ba417d5b2c591dde60cfd1/vaadin-upload-file.html#L63)):*
For JS, the gulp lint throws error for decimals without leading zero. I'd suggest to use 0.3 and 0.1 in CSS also.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-upload/75)*

**Comment by platosha** [comment]
Review status: 1 of 4 files reviewed at latest revision, 4 unresolved discussions.

---

*[vaadin-upload-file.html, line 18 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/75#-KDOc_iKWkXHdoC3S8mw:-KDSJqHxP3Yd99tegaX0:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/8cd665bb511a5a63a05df129430ace0553c466d2/vaadin-upload-file.html#L18)):*
Done.

---

*[vaadin-upload-file.html, line 19 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/75#-KDOdgppj4VvZst2-UQI:-KDSJt3xMAD59b800HCt:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/8cd665bb511a5a63a05df129430ace0553c466d2/vaadin-upload-file.html#L19)):*
Done.

---

*[vaadin-upload-file.html, line 63 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/75#-KDSErPLMmg1BzPg7Vkg:-KDSKoPLs3PtbZXEp07Q:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/8cd665bb511a5a63a05df129430ace0553c466d2/vaadin-upload-file.html#L63)):*
Done.

---

*[vaadin-upload-file.html, line 199 \[r1\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/75#-KDOfXd6dKg0UFxLCQRN:-KDSKpb3aPfq7XH-SFLh:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/8cd665bb511a5a63a05df129430ace0553c466d2/vaadin-upload-file.html#L199)):*
Done.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-upload/75)*

**Comment by manolo** [comment]
A couple of comments in doc, otherwise lgtm

---

Review status: 1 of 4 files reviewed at latest revision, 8 unresolved discussions.

---

*[vaadin-upload-file.html, line 19 \[r2\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/75#-KDSOlG7saj3tVISw-9a:-KDSOlG7saj3tVISw-9b:681498261) ([raw file](https://github.com/vaadin/vaadin-upload/blob/fc976ab7480b844f06a0e6cfcbbb5f8a42400b43/vaadin-upload-file.html#L19)):*
repeating complete and file so many times looks odd to me, I would rephrase it:
A mixin that is applied to the complete status icon when the upload process succeeds.
(also applies to adoc below)

---

*[vaadin-upload-file.html, line 20 \[r2\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/75#-KDSPGSjrmVp4ChZU4WQ:-KDSPGSknUc8CuJlTe-2:1913886193) ([raw file](https://github.com/vaadin/vaadin-upload/blob/fc976ab7480b844f06a0e6cfcbbb5f8a42400b43/vaadin-upload-file.html#L20)):*
Reword: 
A mixin that is applied to the error status icon when the upload process fails.

---

*[vaadin-upload-file.html, line 22 \[r2\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/75#-KDSPSldbR2ThJzIf9zJ:-KDSPSldbR2ThJzIf9zK:-300486253) ([raw file](https://github.com/vaadin/vaadin-upload/blob/fc976ab7480b844f06a0e6cfcbbb5f8a42400b43/vaadin-upload-file.html#L22)):*
when file upload is complete

---

*[vaadin-upload-file.html, line 23 \[r2\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/75#-KDSPczH3pRwjX6W5JXU:-KDSPczH3pRwjX6W5JXV:2058818823) ([raw file](https://github.com/vaadin/vaadin-upload/blob/fc976ab7480b844f06a0e6cfcbbb5f8a42400b43/vaadin-upload-file.html#L23)):*
when an error happens

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-upload/75#-:-KDSQ2tjP7i-HHvQHjVc:-670572336)*

**Comment by manolo** [comment]
Reviewed 1 of 3 files at r2.
Review status: 1 of 4 files reviewed at latest revision, 6 unresolved discussions.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-upload/75)*

**Comment by platosha** [comment]
Review status: 1 of 4 files reviewed at latest revision, 6 unresolved discussions.

---

*[vaadin-upload-file.html, line 19 \[r2\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/75#-KDSOlG7saj3tVISw-9a:-KDSRGlOH17NKXMNBWlF:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/fc976ab7480b844f06a0e6cfcbbb5f8a42400b43/vaadin-upload-file.html#L19)):*
Done.

---

*[vaadin-upload-file.html, line 20 \[r2\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/75#-KDSPGSjrmVp4ChZU4WQ:-KDSRHNVe5fbyCAwWcpm:-499146009) ([raw file](https://github.com/vaadin/vaadin-upload/blob/fc976ab7480b844f06a0e6cfcbbb5f8a42400b43/vaadin-upload-file.html#L20)):*
Done.

---

*[vaadin-upload-file.html, line 22 \[r2\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/75#-KDSPSldbR2ThJzIf9zJ:-KDSRHlI-VSVK-4UNnq7:-369046222) ([raw file](https://github.com/vaadin/vaadin-upload/blob/fc976ab7480b844f06a0e6cfcbbb5f8a42400b43/vaadin-upload-file.html#L22)):*
Done. Mind that the progress-complete mixin is now described in a different way.

---

*[vaadin-upload-file.html, line 23 \[r2\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/75#-KDSPczH3pRwjX6W5JXU:-KDSRKefS3jifT7A81GB:354037885) ([raw file](https://github.com/vaadin/vaadin-upload/blob/fc976ab7480b844f06a0e6cfcbbb5f8a42400b43/vaadin-upload-file.html#L23)):*
Done. Mind that the progress-error mixin is now described in a different way.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-upload/75)*

**Comment by platosha** [comment]
Some minor inconsistency in the descriptions, IMHO. Should we update progress-complete progress-error mixin descriptions in the same way?

---

Review status: 1 of 4 files reviewed at latest revision, 6 unresolved discussions.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-upload/75#-:-KDSRq1Toru3l4tmceP6:-1618002792)*

**Comment by manolo** [comment]
Review status: 1 of 4 files reviewed at latest revision, 6 unresolved discussions.

---

*[vaadin-upload-file.html, line 22 \[r2\]](https://reviewable.io:443/reviews/vaadin/vaadin-upload/75#-KDSPSldbR2ThJzIf9zJ:-KDSXaI2Z9jr9_Wn6DAe:-1348136475) ([raw file](https://github.com/vaadin/vaadin-upload/blob/fc976ab7480b844f06a0e6cfcbbb5f8a42400b43/vaadin-upload-file.html#L22)):*
The meaning is correct in both cases, and repetitive sentences could be boring. 

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-upload/75)*

**Comment by manolo** [comment]
Reviewed 3 of 3 files at r3.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-upload/75)*

**Comment by manolo** [comment]
:lgtm:

---

Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-upload/75#-:-KDSYMKTMA89zo6T7PpX:1417088721)*

**Comment by samiheikki** [comment]
:lgtm:

---

Reviewed 3 of 3 files at r3.
Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/vaadin/vaadin-upload/75#-:-KDSZ27d87o0iMWEVv_M:1417088721)*
